optimade/server/utils.py

```diff
@@ -156,10 +156,21 @@ def get_included_relationships(
 
     included = {}
     for entry_type in endpoint_includes:
-        compound_filter = " AND ".join(
+        compound_filter = " OR ".join(
             ["id={}".format(ref_id) for ref_id in endpoint_includes[entry_type]]
         )
-        params = {"filter": compound_filter}
+        params = EntryListingQueryParams(
```

Comment:
Do you need to pass all of the other parameters in as well? Will it not just take the default values? Or is it because you want to be verbose here for clarity on what's being passed?

Reply:
I had to pass every single one to get it to run, perhaps we can add default values to the `Query` fields in the params, to get around this, but I couldn't see how to last night

Reply:
Yeah, I've had similar problems attempting to make `/links` work properly. Sometimes is passes a `Query` object, other times it insists a parameter is of `str` type.